.npmignore

```diff
@@ -13,22 +13,20 @@
 /_site
 /_layouts
 /_includes
-/.github
+/.*
 
-npm-debug.log
-instance.json
-test-gallery.html
-Gemfile.lock
+_config.yml
+bower.json
+codecov.yml
 gallery.html
+Gemfile.lock
 Gemfile
 nodemon.json
+test-gallery.html
+tsconfig.json
 tslint.json
-bower.json
-_config.yml
-.travis.yml
-.jekyll-metadata
-.vscode
-.editorconfig
+yarn.lock
 vl.sublime-project
 **/*.ts
 !**/*.d.ts
```

Comment:
Just file an issue  #1955 for that. 

The reason I wanna separate this from that PR is that this PR makes the current approach correct.  

Meanwhile, #1955 introduces a different approach.  If we merge this first and have that PR, we can always revert that PR if we find that that approach is actually not better.